HW0/P5/P5.txt

```diff
@@ -1 +1,327 @@
-# Your answers here.
+###############################
+
+# CS 205 Fall 2015 Homework 0 Problem 5
+# Submitted by Kendrick Lo (Harvard ID: 70984997)
+# Github username: ppgmg
+
+Summary
+-------
+
+1. 255 seconds
+2. 34 seconds
+3. 8 seconds
+4. log N seconds
+5. 15 seconds
+6. 272 seconds, 143 seconds
```

Comment:
Thanks for your comment.  I'm not sure I agree with that interpretation though. The question asks what happens if each cashier comes up to you and takes 1 bag. The question also says that you can't hand bags to more than one person at a time. So I'm not sure for the case with 1 bag how it can be faster than 256 seconds, if each cashier can take 1 bag from you, and that individual action takes 1 second long...